src/routes/(console)/apply-credit/+page.svelte

```diff
@@ -227,24 +227,22 @@
     function getBillingPlan(): Tier | undefined {
         const campaignPlan =
             campaign?.plan && $plansInfo.get(campaign.plan) ? $plansInfo.get(campaign.plan) : null;
-        const orgPlan =
-            selectedOrg?.billingPlan && $plansInfo.get(selectedOrg.billingPlan)
-                ? $plansInfo.get(selectedOrg.billingPlan)
-                : null;
+        const newPlan = page.data.plansInfo.get(billingPlan);
 
-        if (!campaignPlan && !orgPlan) {
-            return;
-        }
-        if (!campaignPlan) {
-            return selectedOrg?.billingPlan;
+        // if campaign has a plan and it's higher than the selected new plan
+        if (campaignPlan?.order > newPlan.order) {
```

Comment:
do we need a null check on `newPlan`, just in case?

Reply:
updated!